docs/aspell.en.pws

```diff
@@ -0,0 +1,313 @@
+personal_ws-1.1 en 0 UTF-8
+aab
+abb
+addr
+aditional
+advices
+aG
+anongid
+anonuid
+apache
+armv
+autoindex
+AUTOREV
+bashrc
+bb
+bbappend
+bbclass
+BBFILE
+BBFILES
+bblayers
+BBLAYERS
+bbmanual
+BBPATH
+bce
+bitbake
+BitBake
+bitbaking
+Bitbucket
+blockdiag
+Broadwell
+bs
+bsp
+bzImage
+cd
+checksum
+Chipset
+CHKSUM
+chmod
+chrpath
+CLI
+cmake
+Cmake
+CMake
+CMAKE
+CMakeLists
+codec
+committer
+conf
+CONF
+config
+CONFIG
+coreutils
+cp
+cpp
+cpu
+crosscompiling
+cvs
+CXX
+dac
+DCMAKE
+dee
+dev
+diffstat
+dir
+distro
+Distro
+DISTRO
+DL
+DockerCI
+Dockerfile
+dockerinstall
+doxygen
+dropdown
+env
+ExecStart
+ExecStartPre
+filesystem
+FindSphinx
+fossology
+Fossology
+FOSSology
+fossy
+frontend
+frontends
+fstab
+functionallity
+gcc
+GCC
+gdb
+GDB
+GenericLinuxOsType
+gerrit
+Gerrit
+gitclone
+github
+GitHub
+GitLab
+glibc
+GNOME's
+Gordan
+GTK
+gzip
+hda
+HMI
+hostfwd
+howto
+htm
+html
+http
+HTTPS
+implementers
+init
+initialAdminPassword
+io
+IP
+IVI
+jdk
+jdoe
+jenkins
+jenkinsci
+jnlp
+jnlpUrl
+journalctl
+json
+keepalive
+keygen
+keypair
+keypairs
+kvm
+KVM
+lables
+LCONF
+LD
+libfoo
+libsdl
+libstdc
+libtemplate
+libvirt
+libxml
+LIC
+linenos
+linux
+literalinclude
+littli
+localhost
+login
+lts
+LTS
+mainpage
+makefiles
+Markuš
+maxdepth
+md
+meaningfull
+mem
+metadata
+Metadata
+mkdir
+mmcblk
+MPL
+multi
+multilib
+mylayer
+mymodule
+natively
+newprj
+nfs
+NFS
+nginx
+Nginx
+nic
+oe
+oecore
+OECORE
+OpenEmbedded
+openjdk
+oprofile
+OSS
+passphrase
+Passphrase
+pc
+pelux
+PELUX
+persistency
+pkgconfig
+png
+PostgreSQL
+pre
+preinstalled
+PREMIRRORONLY
+prepopulate
+prerequesits
+presant
+PRIOTIRY
+proj
+PV
+py
+pysqlite
+pytest
+qemu
+QEMU
+qemux
+qmake
+QtCreator
+qxl
+QXL
+rc
+readme
+README
+rebase
+rebasing
+regex
+repo
+Repo
+repos
+repotool
+reproducability
+reproducibility
+roadmap
+rsa
+rsize
+rst
+runnables
+rw
+scp
+screenshots
+sdc
+SDE
+sdk
+SDK
+SDKMACHINE
+SDKTARGETSYSROOT
+sed
+sel
+sendfile
+sftp
+smp
+soundhw
+SPDX
+src
+SRC
+SRCREV
+sstate
+SSTATE
+subdir
+subdirectory
+submodule
+subnet
+substituded
+sudo
+svn
+SWD
+swf
+SWF
+symlink
+sysroot
+SYSROOT
+systemctl
+systemd
+Systemd
+taglist
+tcp
+TCP
+templateconf
+TEMPLATECONF
+templatepublicclass
+texinfo
+timeo
+TimeoutStartSec
+tmp
+toctree
+toctrees
+toolchain
+TOPDIR
+ttyS
+txt
+ui
+UML
+unittest
+unix
+URI
+url
+Url
+usefull
+useradd
+usermod
+username
+Username
+usr
+utils
+uvesafb
+Vagrantfile
+vga
+VirtualBox
+virtualization
+virtulized
```

Comment:
Misspelled word

Reply:
I was planning to do fix them in another commit :)